tests/containers/podman_image.pm

```diff
@@ -17,18 +18,18 @@ use utils;
 use containers::common;
 use containers::container_images;
 use suse_container_urls 'get_suse_container_urls';
-use scheduler 'get_test_suite_data';
+use version_utils qw(get_os_release check_host_os);
 
 sub run {
     my ($image_names, $stable_names) = get_suse_container_urls();
-    my $test_data = get_test_suite_data();
-    my $runtime   = "podman";
-    install_podman_when_needed($test_data->{host_os});
+    my $host_distri = get_os_release;
```

Comment:
you are calling `get_os_release()` here in scalar context. So $host_distri would contain 1,2 or 3 depending how many version parts the `get_os_release()` function found.

Reply:
nice catch. somehow i miss this. thanks